src/navigation/navigation-item.js

```diff
@@ -0,0 +1,71 @@
+const component = {};
+
+/**
+  * @ngdoc directive
+  * @name bdNavigationItem
+  * @module buildium.angular-elements.navigation
+  * @restrict E
+  * 
+  * @param {Boolean} isActive
+  * @param {Boolean} isDisabled
+  * 
+  * @example
+    <example name="bd-navigation" module="buildium.angular-elements.navigation">
+        <file name="index.html">
+            <div class="page-content">
+                <bd-navigation>
+                    <bd-navigation-item>
+                        <bd-navigation-link>
+                            <navigation-title>Settings</navigation-title>
+                        </bd-navigation-link>
+                        <navigation-menu>
+                            <bd-navigation>
+                                <bd-navigation-item>
+                                    <bd-navigation-link>
+                                        <navigation-title>Billing</navigation-title>
+                                    </bd-navigation-link>
+                                </bd-navigation-item>
+                            </bd-navigation>
+                        </navigation-menu>
+                    </bd-navigation-item>
+                </bd-navigation>
+            </div>
+        </file>
+    </example>
+  *
+  */
+
+component.bindings = {
+    isActive: '<?',
+    isDisabled: '<?'
+};
+
+component.template = `
+<div bd-accordion-toggle 
+    disabled="$ctrl.isDisabled"
+    is-open="$ctrl.isActive"
+    on-change="$ctrl.onAccordionToggleChange(isAccordionGroupOpen)"
+    ng-class="['navigation__item', {'navigation__item--active': $ctrl.isActive, 'navigation__item--disabled': $ctrl.isDisabled}]">
```

Comment:
does this mean "navigation__item" is always added? If so, why not just use a class attribute?

Reply:
yes, I was seeing behavior where the class was being removed when added to `class`. I'll test this again before merging

Reply:
weird